.gitignore

```diff
@@ -1,16 +1,15 @@
-# Composer
+# IDE and local environment
+/.idea/
```

Comment:
Keep "OS" and "IDE and local environment" patterns on your local machine in user home
`git config --global core.excludesFile ~/.gitignore`
A single .gitignore file cannot cover every OS, every IDE and every local environment.
Please read https://git-scm.com/docs/gitignore

Reply:
This will effect every project of yours and put an end to this problem.

Reply:
What do you mean by this? The .idea folder is still here, right? And the intention is not that I don't commit them, it's so other peopel don't commit these files. ;)

Reply:
> it's so other peopel don't commit these files. ;)

Tough questions. Everyone should have a `core.excludesFile`
The other solution it to have a 30 000 000 000 line long gitignore file covering temporary OS files of a smart microwave oven.

Reply:
> other peopel don't commit these files

That may be called _pull request review_.